apps/webapp/web/static/js/index.js

```diff
@@ -34,17 +34,41 @@ function requireAuth(nextState, replace) {
   }
 }
 
-ReactDOM.render(
-  <Router history={browserHistory}
-    render={applyRouterMiddleware(useRelay)}
-    environment={() => Auth.getEnvironment()}>
-    <Route path="/" component={Hello}/>
+const routes = (
+  <Route path="/" >
+    <IndexRoute component={Hello}/>
     <Route path="/login" component={Login}/>
-    <Route path="/admin" component={AdminLayout} onEnter={requireAuth}>
+    <Route path="/admin" component={AdminLayout} onEnter={requireAuth} >
       <IndexRoute component={Hello}/>
       <Route path="star-wars" component={StarWarsApp} queries={StarWarsQueries}/>
       <Route path="graphiql" component={GraphiQL} />
     </Route>
-  </Router>,
+  </Route>
+)
+
+class Application extends React.Component{
```

Comment:
If you want to make this file a little bit more _cleaner_ then you might want to move this `class` to its own file **app.js**. 

Anyway, I don't mind having this class here.